blur bg on err pages

maria-vslvn opened the pull request:
# Summary

Fixes #1485 

reproduce an err page an check the background

  # To Test

1. Open the page[ "Cow Game" ](https://pr1829--gpswapui.review.gnosisdev.com/#/play/cow-runner)to throw an error so we can see the app crashing.



![image](https://user-images.githubusercontent.com/622217/141478074-a8a6d518-4436-4a93-a1a1-464f788ead3e.png)

alongoni reviewed (changes requested):
Looks great, but we should add the Cowswap logo to give users an option to go back to home.

I've worked on this solution some time ago. Please check the [PR#1259](https://github.com/gnosis/cowswap/pull/1259). And especially the way we include the logo here:
https://github.com/gnosis/cowswap/pull/1259/commits/a1d52105b3a6b16c737506259ff8d165ed8f97ee

elena-zh commented:
Hi @maria-vslvn , how can I generate a hard error from the UI part in order to test the changes?

maria-vslvn commented:
@elena-zh @alongoni some changes are pushed, now the bug can be reproduced by trying to play the runner-cow game

elena-zh reviewed (changes requested):
Changes LGTM besides one issue: I can't scroll down the page to see links at the bottom of the error section
(I was able to reproduce the issue in Firefox)
![image](https://user-images.githubusercontent.com/70885163/141492907-3083f738-f114-41f2-9a06-d41d8ef41d19.png)

anxolin commented:
The change looks good, but it also happens to me the issue with the scroll

alongoni commented:
Hey @maria-vslvn I've found this: (it's a class called `noScroll`)
![image](https://user-images.githubusercontent.com/622217/141995303-d8fd538c-6205-4af4-9b59-22e9e7e99895.png)
![image](https://user-images.githubusercontent.com/622217/141995115-20109342-d288-43d4-b711-c1f4fb8dc562.png)

maria-vslvn commented:
@elena-zh @W3stside i added footer, and removed the noScroll in the view, previously i asked for help in fixing the script, that  existed there and added the class for all views, so i can close it if there is no need in this?

elena-zh commented:
Hey @maria-vslvn , changes LGTM! However, it would be great to update the branch with the latest version 1.9.0, due to 1.4.1 is here

elena-zh commented:
Changes LGTM. 
The only issue I can mention is that the page has become wider than usual content pages
![image](https://user-images.githubusercontent.com/70885163/151001050-3b67ff5e-a755-4933-8dc1-5dcbf6216809.png)
 Compare with 'About' page
![image](https://user-images.githubusercontent.com/70885163/151001184-7826fdbb-a2ef-4530-8c25-83847e6816ac.png)


@alfetopito , are we OK with this?

alfetopito commented on src/custom/pages/games/CowRunner/index.tsx:
```diff
@@ -29,6 +29,10 @@ const Wrapper = styled(Page)`
 `
 
 export default function CowRunnerPage() {
+  if (0 === 0) {
+    throw new Error('This cow is dropping some serious milk 🥛!!')
+  }
+
```
Please remember to remove this before merging!

elena-zh commented:
@maria-vslvn , could you please revert the change that forces a hard error on the CowGame page? 
Then we can merge this PR into Develop

elena-zh commented:
@maria-vslvn . @ramirotw , could we please merge this?
